lib/identity/client.js

```diff
@@ -155,6 +155,27 @@ class Client extends PartialClient {
     return resultQuery
   }
   /**
+   * getLatestSecrets
+   *
+   * @param {string} secretName   The name of the Secret given by User.
+   * @param {string} secretType   The type of the Secret chosen by User.
+   *
+   *
+   * @return {Promise<Record>}
+   */
+  async getLatestSecret(secretName, secretType) {
+    const request = new Search(true)
+    let searchType = `tozny.secret.${SECRET_UUID}.${secretType}.${secretName}`
+    request.match({ type: searchType })
+    let resultQuery = await this.storage.search(request)
+    let resultList = await resultQuery.next()
+    let lengthOfResults = resultList.length
+    if (resultList.length != 0) {
+      return resultList[lengthOfResults - 1]
+    }
+    return resultList[0]
```

Comment:
so I guess the caller of this code just always has to check this response is not null? Can we return a struct that has a field for if the secret exists at all, and a field for holding that secret (if it does exist)